plugins/ctx.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ctx
+spec:
+  homepage: https://github.com/ahmetb/kubectx
+  shortDescription: Switch between clusters in your kubeconfig
+  version: v0.7.1
+  description: |
+    Also known as "kubectx", a utility to switch between context entries in
+    your kubeconfig file effectively.
```

extreme nit-picking:
```suggestion
    your kubeconfig file efficiently.
```